### PR TITLE
i18n(zh-cn): Update `upgrade-astro.mdx`

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Update `upgrade-astro.mdx`

#### Related issues & labels (optional)

- #7652

### CODE REVIEW

File: src/content/docs/zh-cn/upgrade-astro.mdx [hunk 1]
```diff
@@ -179,3 +179,25 @@ Astro 尽可能地遵循 [语义化版本控制](https://semver.org/)，这是
   一旦这些功能结束了它们的实验期，它们将遵循正常的语义化版本控制合约。
 
 - **文档的改进**（例如，参考和错误消息）。它们是为 `docs` 仓库从源代码构建的。这允许 Astro 在文档源内容存储在主 `astro` 仓库的情况下，快速更新文档修复和改进。
+
+### Node.js 支持和升级策略
+
+#### 支持
+
+- Astro 支持 [**latest _Maintenance_ LTS** 版本的 Node.js](https://nodejs.org/en/about/previous-releases#release-schedule)。
+- Astro 支持 [**current _Active_ LTS** 版本的 Node.js](https://nodejs.org/en/about/previous-releases#release-schedule)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/upgrade-astro.mdx:188]:
```suggestion
- Astro 支持 [**当前的 _Active_ LTS** 版本的 Node.js](https://nodejs.org/en/about/previous-releases#release-schedule)。
```

File: src/content/docs/zh-cn/upgrade-astro.mdx [hunk 2]
```diff
@@ -179,3 +179,25 @@ Astro 尽可能地遵循 [语义化版本控制](https://semver.org/)，这是
   一旦这些功能结束了它们的实验期，它们将遵循正常的语义化版本控制合约。
 
 - **文档的改进**（例如，参考和错误消息）。它们是为 `docs` 仓库从源代码构建的。这允许 Astro 在文档源内容存储在主 `astro` 仓库的情况下，快速更新文档修复和改进。
+
+### Node.js 支持和升级策略
+
+#### 支持
+
+- Astro 支持 [**latest _Maintenance_ LTS** 版本的 Node.js](https://nodejs.org/en/about/previous-releases#release-schedule)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/upgrade-astro.mdx:187]:
```suggestion
- Astro 支持 [**最近的 _Maintenance_ LTS** 版本的 Node.js](https://nodejs.org/en/about/previous-releases#release-schedule)。
```

**Review by liruifengv — APPROVED** [review]
LGTM

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**